package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@guardian/image-rendering",
-  "version": "6.0.0",
+  "version": "7.0.0",
   "description": "Handles parsing images from CAPI and rendering them in the *-rendering projects",
   "main": "dist/index.js",
-  "dependencies": {
+  "peerDependencies": {
     "@emotion/babel-plugin": "^11.1.2",
```

Comment:
@JamieB-gu This is more inline with how `atoms-rendering` and `discussion-rendering` do it. It avoids the problem we had when we'd have two conflicting versions installed in the host app because `peerDependencies` was specifying a different version than `dependencies`. Adding it to `devDependencies` too allows us to build the project locally and run storybook. I'm not aware of downsides as I haven't encountered any so maybe there's a reason it was in `dependencies` in the first place.